firmware/src/rdm_responder.c

```diff
@@ -132,14 +116,14 @@ int RDMResponder_HandleDUBRequest(const uint8_t *param_data,
 void RDMResponder_BuildHeader(const RDMHeader *incoming_header,
                               RDMResponseType response_type,
                               RDMCommandClass command_class,
-                              RDMPid pid,
+                              uint16_t pid,
```

Comment:
Do we need to change this, is it because our manufacturer PIDs aren't in the enum?